t/opbasic/arith.t

```diff
@@ -2,8 +2,12 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    require './test.pl';
-    set_up_inc('../lib');
+    # we can't use test.pl here (see the comment below)
+    #require './test.pl';
+    #set_up_inc('../lib');
+    # so inline set_up_inc() and its call to is_miniperl
+    @INC = () if defined DynaLoader::boot_DynaLoader;
```

Comment:
Excuse me for cutting in, but I think `&` is missing here:
```suggestion
    @INC = () if defined &DynaLoader::boot_DynaLoader;
```

Reply:
Thanks, fixed.